WebPhapp/client/src/App/pages/PrescriptionEdit.js

```diff
@@ -74,126 +74,149 @@ Fields:
 - Drug ID        - Can't change
 - Filled Dates   - If filled, cannot change
 - Written Date   - Static
-- quantity       - Changable?
+- Quantity       - Changable?
```

Comment:
These were some of my personal notes that I didn't remove... Could you remove these for me?

Reply:
removed